From bcaad43b797db32b2860b1325715211c2eded8be Mon Sep 17 00:00:00 2001
From: Ken Ma <make@marvell.com>
Date: Wed, 15 Mar 2017 01:25:27 +0800
Subject: [PATCH 0885/1345] dts: a3700: force xhci reset after resume

commit  7fc13e687eb191f24e3cbd19271287e89b39816e from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

- Armada 3700 must force XHCI reset after resume, otherwise its XHCI
  reset will get stuck so add the property "needs-reset-on-resume" to
  its usb3 node.

Change-Id: I72bd6267beffbd12bbc654427627a79de42c627c
Signed-off-by: Ken Ma <make@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/37444
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Victor Gu <xigu@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index 0d9eb46..5a4c729 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -389,6 +389,7 @@
 				interrupts = <GIC_SPI 3 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&sb_periph_clk 12>;
 				separated-phys-for-usb2-usb3;
+				needs-reset-on-resume;
 				status = "disabled";
 			};
 
-- 
1.7.9.5

